go.mod

```diff
@@ -13,11 +13,11 @@ require (
 	github.com/cespare/xxhash/v2 v2.1.1
 	github.com/coreos/go-oidc/v3 v3.0.0
 	github.com/envoyproxy/go-control-plane v0.9.9-0.20201210154907-fd9021fe5dad
-	github.com/envoyproxy/protoc-gen-validate v0.1.0
+	github.com/envoyproxy/protoc-gen-validate v0.4.1
```

Comment:
\cc @wasaga this seems like a big jump from the initial commit. Was pinning to v0.1.0 intentional?